components/LinkSummary.tsx

```diff
@@ -0,0 +1,28 @@
+import { FC } from 'react'
+import Link from 'next/link'
+
+export interface LinkSummaryItem {
+  href: string
+  text: string
+}
+
+export interface LinkSummaryProps {
+  id: string
+  links: LinkSummaryItem[]
+}
+
+const LinkSummary: FC<LinkSummaryProps> = ({ id, links }) => {
+  return (
+    <section id={id}>
```

Comment:
I'm not sure the id is adding any value to this component...

Reply:
My original thought process was maybe for some form of accessibility purpose or incase we needed to get the entire section by id for something but if it's not necessary I will remove it